Meridian59/Data/Models/AdminData/AdminObjectProperty.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminObjectProperty : INotifyPropertyChanged
+    {
+        public string PropertyName { get; set; }
+        public string PropertyType { get; set; }
+        
+        private string _propertyValue { get; set; }
+        public string PropertyValue {
+            get { return _propertyValue; }
+            set
+            {
+                _propertyValue = value;
```

Comment:
Yea my workaround for this binding loop resulted in some other code you have already commented on. I'll fix it. Thanks.